app/helpers/dropdown_selection_helper.rb

```diff
@@ -6,4 +6,20 @@ def chosen_user
   def chosen_period
```

Comment:
are these `period` methods dead? I mean, can we remove them or are they still in use?